prometheus-operator/.gitignore

```diff
@@ -0,0 +1 @@
+uninstall.sh
```

Comment:
If you have an uninstall.sh (that takes no parameters), please feel free to commit it - this is exactly the mechanism we are going to be using to uninstall applications, other than just reversing the `kubectl apply -f app.yaml` if one exists.